polaris-react/src/styles/shared/_interaction-state.scss

```diff
@@ -6,7 +6,7 @@
   left: calc(-1 * var(--p-space-2));
   height: 100%;
   display: block;
-  width: var(--p-border-width-3);
+  width: 3px;
```

Comment:
This is a very small border to the right of a selected item. Similarly to others I would expect this to be connected to border tokens and if the the 3px value changed I would expect this to be linked.